lib/mayaUsd/ufe/UsdTransform3dMatrixOp.cpp

```diff
@@ -355,6 +355,18 @@ Ufe::ScaleUndoableCommand::Ptr UsdTransform3dMatrixOp::scaleCmd(double x, double
         UsdTimeCode::Default());
 }
 
+Ufe::SetMatrix4dUndoableCommand::Ptr UsdTransform3dMatrixOp::setMatrixCmd(const Ufe::Matrix4d& m)
+{
+    return nullptr;
```

Comment:
parent -absolute won't work for matrix op prims yet, but implementing this should be trivial (see setMatrix()) below.